simpleclient/src/main/java/io/prometheus/client/SimpleTimer.java

```diff
@@ -0,0 +1,70 @@
+package io.prometheus.client;
+
+import static io.prometheus.client.Collector.NANOSECONDS_PER_SECOND;
+
+/**
+ * SimpleTimer, to measure elapsed duration as secondas a double.
+ *
+ * <p>
+ * This is a helper class intended to measure latencies and encapsulate the conversion to seconds without losing precision.
+ *
+ * <p>
+ * Example usage:
+ * <pre>
+ * {@code
+ *   class YourClass {
+ *     static final Summary requestLatency = Summary.build()
+ *         .name("requests_latency_seconds")
+ *         .help("Request latency in seconds.")
+ *         .labelNames("aLabel")
+ *         .register();
+ *
+ *     void processRequest(Request req) {
+ *        SimpleTimer requestTimer = new SimpleTimer();
+ *        try {
+ *          // Your code here.
+ *        } finally {
+ *          requestTimer.labels("aLabelValue").observe(requestTimer.elapsedSeconds());
+ *        }
+ *     }
+ *   }
+ * }
+ * </pre>
+ *
+ * Keep in mind that preferred approaches avoid using this mechanism if possible, since latency labels should be minimized and
```

Comment:
I'd put this up above the example, as it won't be read otherwise.

Latency labels would imply the latency is going in the label. I'd say latency metrics broken out by outcome.